### PR TITLE
Fix to display tram services in drop down

### PR DESCRIPTION
**Pull request by alvak1**
Create service consequence page should now display tram services in drop down when vehicle mode is selected. This is extended to all other vehicle mode values.

### CODE REVIEW

File: site/data/refDataApi.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const fetchStops = async (input: FetchStopsInput) => {
 interface FetchServicesInput {
     adminAreaCodes?: string[];
     dataSource?: Modes;
```
Comment (alainajaffer) [inline site/data/refDataApi.ts:47]: could we rename this interface to DataSource

File: site/data/refDataApi.ts [hunk 2]
```diff
@@ -45,6 +45,7 @@ export const fetchStops = async (input: FetchStopsInput) => {
...
+    modes?: string[];
```
Comment (alainajaffer) [inline site/data/refDataApi.ts:48]: Then we can make this an interface called Mode with the transport values

File: site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx [hunk 3]
```diff
@@ -244,18 +244,34 @@ const CreateConsequenceServices = (props: CreateConsequenceServicesProps): React
         }
     }, [pageState?.inputs?.services]);
 
+    const fetchData = async (source: Modes, vehicleMode: string) => {
+        let mode: string[] = [];
+        if (vehicleMode === "ferryService") {
+            mode = ["ferry"];
+        } else if (vehicleMode === "tram") {
```
Comment (alainajaffer) [inline site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx:251]: as above

File: site/data/refDataApi.ts (hunk 1 above)
Comment (alvak1) [reply site/data/refDataApi.ts]: done

File: site/data/refDataApi.ts (hunk 2 above)
Comment (alvak1) [reply site/data/refDataApi.ts]: done

File: site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx [hunk 4]
```diff
@@ -244,18 +244,34 @@ const CreateConsequenceServices = (props: CreateConsequenceServicesProps): React
         }
     }, [pageState?.inputs?.services]);
 
+    const fetchData = async (source: Modes, vehicleMode: string) => {
+        let mode: string[] = [];
+        if (vehicleMode === "ferryService") {
```
[the remark this answers is not in the document]
Comment (alvak1) [reply site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx]: done

File: site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx (hunk 3 above)
Comment (alvak1) [reply site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx]: done

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**